pkg/controller/csr_check.go

```diff
@@ -207,7 +221,11 @@ func authorizeCSR(
 		}
 		// The CSR requested a DNS name that did not belong to the machine
 		if !foundSan {
-			return fmt.Errorf("DNS name '%s' not in machine names: %s", san, strings.Join(attemptedAddresses, " "))
+			//TODO: set annotation/emit event here.
+			// return error so we requeue, in case machine network is out of date
+			// for some reason
+			klog.Errorf("%v: DNS name '%s' not in machine names: %s", req.Name, san, strings.Join(attemptedAddresses, " "))
```

Comment:
Are there security implications of logging things such as DNS names?

Reply:
No.  DNS names appear in non-secret objects all over the place.  The public cert is just that, public.